src/index.html

```diff
@@ -13,6 +13,48 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image"></div>
+
+      <div class="card__info">
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <h3 class="card__code">Product code: 195434</h3>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
```

Comment:
According to the checklist, BEM methodology should be used for class names. The class 'stars--4' suggests a modifier, but the 'stars__star' class does not reflect that it's a part of 'stars--4'. Ensure that the classes are consistent with BEM.